tests/Integration/Behaviour/Features/Scenario/Search/add_alias.feature

```diff
@@ -0,0 +1,101 @@
+@restore-aliases-before-feature
+@clear-cache-before-feature
+@clear-cache-after-feature
+@add
+Feature: Add basic alias from Back Office (BO)
+  As a BO user
+  I need to be able to add new alias with basic information from the BO
+
+  Scenario: I add an alias with basic information
+    Given following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+    When I add alias with following information:
+      | alias   | large |
+      | search  | big   |
+    Then following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+      | alias3       | large  | big    |
+
+  Scenario: I add a multiple aliases with basic information
+    Given following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+      | alias3       | large  | big    |
+    When I add alias with following information:
+      | alias   | bliu,blu |
+      | search  | blue     |
+    Then following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+      | alias3       | large  | big    |
+      | alias5       | bliu   | blue   |
+      | alias6       | blu    | blue   |
+
+  Scenario: I add a multiple aliases with duplicate search field
+    Given following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+      | alias3       | large  | big    |
+      | alias5       | bliu   | blue   |
+      | alias6       | blu    | blue   |
+    When I add alias with following information:
+      | alias   | bliu,sapphire |
+      | search  | blue          |
+    Then following aliases should exist:
+      | id reference | alias    | search |
+      | alias1       | bloose   | blouse |
+      | alias2       | blues    | blouse |
+      | alias3       | large    | big    |
+      | alias5       | bliu     | blue   |
+      | alias6       | blu      | blue   |
+      | alias7       | sapphire | blue   |
+
+    When I add alias with following information:
+      | alias   | blah |
+      | search  | blue |
+    Then following aliases should exist:
+      | id reference | alias    | search |
+      | alias1       | bloose   | blouse |
+      | alias2       | blues    | blouse |
+      | alias3       | large    | big    |
+      | alias5       | bliu     | blue   |
+      | alias6       | blu      | blue   |
+      | alias7       | sapphire | blue   |
+      | alias8       | blah     | blue   |
+
+  Scenario: I add alias with empty alias field
+    When I add alias with following information:
+      | alias   |      |
+      | search  | blue |
+    Then following aliases should exist:
```

Comment:
In command level I think you should assert that aliases are not empty.
If they are, throw Core/InvalidArgumentException.
So here as well the step should assert the same (surround command with try catch and setLastException), then implement a dedicated step to check that last exception was that InvalidArgumentException and finally check that no aliases were created